src/libsystemd/sd-bus/bus-common-errors.h

```diff
@@ -59,6 +59,7 @@
 #define BUS_ERROR_SLEEP_VERB_NOT_SUPPORTED     "org.freedesktop.login1.SleepVerbNotSupported"
 #define BUS_ERROR_SESSION_BUSY                 "org.freedesktop.login1.SessionBusy"
 #define BUS_ERROR_NOT_YOUR_DEVICE              "org.freedesktop.login1.NotYourDevice"
+#define BUS_ERROR_MAINT_TIME_NOT_SCHEDULED     "org.freedesktop.login1.DesignatedMaintenanceTimeNotScheduled"
```

Comment:
please name the C identifier with the same words as the error string. i.e. BUS_ERROR_DESIGNATED_MAINTENANCE_TIME_NOT_SCHEDULED.